jest/authSlice.spec.ts

```diff
@@ -0,0 +1,32 @@
+import "jest";
+import { login, logout, authReducer } from "../src/GlobalRedux/store";
+import { AuthState } from "../src/types/types";
+import { expect } from "@jest/globals";
+
+const wallet = "SBKQYX26DP6GLTVKX6SADEUO3W3WC7GUQGXDINGM2VIHGHNW5KMB5SB6";
+
+describe("Authentication Redux Store", () => {
+  it("should handle login action adding login credentials to store", () => {
```

Comment:
Let's capitalize our tests `Should...` that way we keep consistency with our other tests.